collections/without_all_test.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+import { assertEquals } from "../testing/asserts.ts";
+import { withoutAll } from "./without_all.ts";
+
+function withoutAllTest<I>(
+  input: Array<I>,
+  excluded: Array<I>,
+  expected: Array<I>,
+  message?: string,
+) {
+  const actual = withoutAll(input, excluded);
+  assertEquals(actual, expected, message);
+}
+
+Deno.test({
+  name: "[collections/withoutAll] empty input",
+  fn() {
+    withoutAllTest([], [], []);
+  },
+});
+
```

Comment:
We should add a test to check against the bug mentioned above

```suggestion
Deno.test({
    name: "[collection/withoutAll] leaves duplicate elements",
    fn () {
        withoutAllTest(new Array(110).fill(3), [1], new Array(110).fill(3));
    },
});
```